package.json

```diff
@@ -36,7 +37,8 @@
     "chai-bn": "^0.1.1",
     "ethjs-abi": "^0.2.1",
     "semver": "^5.6.0",
-    "truffle-contract": "^4.0.8"
+    "truffle-contract": "^4.0.8",
+    "web3-utils": "^1.0.0-beta.55"
```

Comment:
Is web3-utls versioned along web3? If so, I'd target beta.37, and check for that one.

Reply:
I don't really see why we should pin to beta 37, but perhaps it's a good idea just in case.